src/parser.js

```diff
@@ -153,6 +153,8 @@ export default class Parser {
           this.handleSwaggerPath(pathValue, href);
         });
 
+        this.handleSwaggerVendorExtensions(this.api, swagger.paths);
```

Comment:
I can't find a better place to attach paths vendor extensions. So I'm adding it to the API category itself, just like the swagger info vendor extensions below (line 257).

I'm worried this might clash, tools would need to understand both extensions may go to the same place. Any thoughts on a better place?